docs/user_guides/fs/feature_group/data_types.md

```diff
@@ -143,6 +143,77 @@ The byte size of each column is determined by its data type and calculated as fo
 | BLOB                            | 256          |
 | other                           | 8            |
 
+
+#### Pre-insert schema validation for online feature groups
+For online enabled feature groups, the dataframe to be ingested needs to adhere to the online schema definitions. The input dataframe is validated for schema checks accordingly.
+The validation is enabled by setting below property when calling `insert()`
+=== "Python"
+    ```python
+    feature_group.insert(df, validation_options={'run_validation':True})
+    ```
+The most important validation checks or error messages are mentioned below along with possible corrective actions. 
+
+1. Primary key contains null values 
+
+    - **Rule** Primary key column should not contain any null values.
+    - **Example correction** Drop the rows containing null primary keys. Alternatively, find the null values and assign them an unique value as per preferred strategy for data imputation.
+        
+        === "Pandas"
+        ```python
+        # Assuming 'id' is the primary key column
+        df = df.dropna(subset=['id'])
+        # For composite keys
+        df = df.dropna(subset=['id1', 'id2'])
+        ```
+
+2. Primary key column missing
+
+    - **Rule** The dataframe to be inserted must contain all the columns defined as primary key(s) in the feature group.
+    - **Example correction** Add all the primary key columns in the dataframe.
+        
+        === "Pandas"
+        ```python
+        # Add missing primary key column
+        df['id'] = some_value
+        # If primary key is an auto-incrementing
+        df['id'] = range(1, len(df) + 1)
+        ```
+
+3. String length exceeded
```

Comment:
I asked already to point to the docs to enlarge the table (which will create a new version of the FG)